conda-forge.yml

```diff
@@ -1,6 +1,4 @@
 build_platform:
-  linux_aarch64: linux_64
-  linux_ppc64le: linux_64
```

Comment:
Cross-compiling doesn't work here atm. Please see PR ( https://github.com/conda-forge/implicit-feedstock/pull/85 ) for a 1st attempt